restart.sh

```diff
@@ -1,39 +0,0 @@
-#!/bin/sh
-
-#paths to kafka and flink. Please adapt according to your configuration.
-FLINK_PATH=/opt/flink
-KAFKA_PATH=/opt/kafka
-
-"$KAFKA_PATH"/bin/kafka-topics.sh --delete --zookeeper localhost:2181 --topic marketSituation
-"$KAFKA_PATH"/bin/kafka-topics.sh --delete --zookeeper localhost:2181 --topic cumulativeRevenueBasedMarketshare
-"$KAFKA_PATH"/bin/kafka-topics.sh --delete --zookeeper localhost:2181 --topic updateOffer
-"$KAFKA_PATH"/bin/kafka-topics.sh --delete --zookeeper localhost:2181 --topic cumulativeRevenueBasedMarketshareHourly
-"$KAFKA_PATH"/bin/kafka-topics.sh --delete --zookeeper localhost:2181 --topic cumulativeRevenueBasedMarketshareDaily
-"$KAFKA_PATH"/bin/kafka-topics.sh --delete --zookeeper localhost:2181 --topic cumulativeTurnoverBasedMarketshare
-"$KAFKA_PATH"/bin/kafka-topics.sh --delete --zookeeper localhost:2181 --topic cumulativeTurnoverBasedMarketshareHourly
-"$KAFKA_PATH"/bin/kafka-topics.sh --delete --zookeeper localhost:2181 --topic cumulativeTurnoverBasedMarketshareDaily
-"$KAFKA_PATH"/bin/kafka-topics.sh --delete --zookeeper localhost:2181 --topic cumulativeAmountBasedMarketshare
-"$KAFKA_PATH"/bin/kafka-topics.sh --delete --zookeeper localhost:2181 --topic cumulativeAmountBasedMarketshareHourly
-"$KAFKA_PATH"/bin/kafka-topics.sh --delete --zookeeper localhost:2181 --topic cumulativeAmountBasedMarketshareDaily
-"$KAFKA_PATH"/bin/kafka-topics.sh --delete --zookeeper localhost:2181 --topic revenue
-"$KAFKA_PATH"/bin/kafka-topics.sh --delete --zookeeper localhost:2181 --topic buyOffer
-"$KAFKA_PATH"/bin/kafka-topics.sh --delete --zookeeper localhost:2181 --topic buyOffers
-"$KAFKA_PATH"/bin/kafka-topics.sh --delete --zookeeper localhost:2181 --topic producer
-"$KAFKA_PATH"/bin/kafka-topics.sh --delete --zookeeper localhost:2181 --topic revenuePerMinute
-"$KAFKA_PATH"/bin/kafka-topics.sh --delete --zookeeper localhost:2181 --topic profitPerMinute
-"$KAFKA_PATH"/bin/kafka-topics.sh --delete --zookeeper localhost:2181 --topic revenuePerHour
-"$KAFKA_PATH"/bin/kafka-topics.sh --delete --zookeeper localhost:2181 --topic profitPerHour
-
-sudo service loggerapp restart
```

Comment:
Why remove the file?

Reply:
In general, are we sure that these files can safely be removed? Sure nobody uses them?

Reply:
`restart.sh`: I recommend restarting the whole platform from a clean state. In the main repo readme are the instructions for that (`docker-compose down`)
`start-jobs.sh`: is done by start-jobmanager.sh
`start-taskmanager.sh`: With the change to the official flink container we no longer need a custom start script for the taskmanager